samples/latest/HelloMvc/NuGet.Config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v2" />
```

Comment:
Ideally this should stay `aspnetvnext`... the samples here should reflect the last overall good build. Though right now we're in the unfortunate state where we haven't had good builds...